src/app/zap-templates/zcl/data-model/chip/joint-fabric-datastore-cluster.xml

```diff
@@ -0,0 +1,370 @@
+    </command>
+
+    <command source="client" code="0x0B" name="UpdateNode" isFabricScoped="true">
+      <description>The command SHALL be used to update the friendly name for a node in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="NodeID" type="node_id"/>
+      <arg id="1" name="FriendlyName" type="char_string" length="32"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x0C" name="RemoveNode" isFabricScoped="true">
+      <description>This command SHALL be used to remove a node from the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="NodeID" type="node_id"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x0D" name="UpdateEndpointForNode" isFabricScoped="true">
+      <description>This command SHALL be used to update the state of an endpoint for a node in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="EndpointID" type="endpoint_no"/>
+      <arg id="1" name="NodeID" type="node_id"/>
+      <arg id="2" name="FriendlyName" type="char_string" length="32"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x0E" name="AddGroupIDToEndpointForNode" isFabricScoped="true">
+      <description>This command SHALL be used to add a Group ID to an endpoint for a node in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="NodeID" type="node_id"/>
+      <arg id="1" name="EndpointID" type="endpoint_no"/>
+      <arg id="2" name="GroupID" type="group_id"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x0F" name="RemoveGroupIDFromEndpointForNode" isFabricScoped="true">
+      <description>This command SHALL be used to remove a Group ID from an endpoint for a node in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="NodeID" type="node_id"/>
+      <arg id="1" name="EndpointID" type="endpoint_no"/>
+      <arg id="2" name="GroupID" type="group_id"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x10" name="AddBindingToEndpointForNode" isFabricScoped="true">
+      <description>This command SHALL be used to add a binding to an endpoint for a node in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="NodeID" type="node_id"/>
+      <arg id="1" name="EndpointID" type="endpoint_no"/>
+      <arg id="2" name="Binding" type="DatastoreBindingTargetStruct"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x11" name="RemoveBindingFromEndpointForNode" isFabricScoped="true">
+      <description>This command SHALL be used to remove a binding from an endpoint for a node in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="ListID" type="int16u"/>
```

Comment:
Spec issue: ListID is defined as:

> ListID represents the unique identifier for the binding to be removed from the endpoint.

but what does this number mean?  What list is it referring to, if any?  Bindings don't generally have "unique identifiers".

Reply:
Tracked by https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11620

Reply:
This id uniquely identifies a DatastoreEndpointBindingEntryStruct entry in the datastores list of these(EndpointBindingList attribute).

Reply:
The spec change will be tracked using [issue #11620](https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11620). Once the spec update is complete, a follow-up SDK PR can be submitted to align it with the updated specification.